sqlmesh/dbt/loader.py

```diff
@@ -42,8 +72,9 @@ def sqlmesh_config(
     project_root = project_root or Path()
     context = DbtContext(project_root=project_root)
     profile = Profile.load(context, target_name=dbt_target_name)
-    model_defaults = kwargs.pop("model_defaults", ModelDefaultsConfig())
-    model_defaults.dialect = profile.target.dialect
+    model_defaults = get_default_dialect(
```

Comment:
Following up on the above. All we need here is the model's default dialect if specified in sqlmesh.